<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>527</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>789</width>
    <height>529</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color:#272741;
}
#frameContainer{
background: #4A4A6C;
}
#widget{
background-color:#272741;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="frameContainer">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QFrame" name="frame">
               <property name="maximumSize">
                <size>
                 <width>1000</width>
                 <height>1000</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QScrollArea" name="scrollArea">
                  <property name="styleSheet">
                   <string notr="true">QLabel{
color:white;
font-size:14px;
}</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="lineWidth">
                   <number>0</number>
                  </property>
                  <property name="verticalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="horizontalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="sizeAdjustPolicy">
                   <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="mainArea">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>637</width>
                     <height>444</height>
                    </rect>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">#mainArea{
background-color:#4A4A6C;
border:5px solid #4A4A6C;
font-size:14px;
;
}</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="1" column="0">
                     <widget class="QFrame" name="frame_4">
                      <property name="maximumSize">
                       <size>
                        <width>301</width>
                        <height>1000</height>
                       </size>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_4">
                       <item>
                        <widget class="QLabel" name="label_5">
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>167</height>
                          </size>
                         </property>
                         <property name="cursor">
                          <cursorShape>PointingHandCursor</cursorShape>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="pixmap">
                          <pixmap>images/image 1.jpg</pixmap>
                         </property>
                         <property name="scaledContents">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                        <widget class="QLabel" name="label_6">
                         <property name="text">
                          <string>San Felipe 3</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QFrame" name="frame_5">
                      <property name="maximumSize">
                       <size>
                        <width>301</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_8">
                       <item>
                        <widget class="QLabel" name="label_7">
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>167</height>
                          </size>
                         </property>
                         <property name="cursor">
                          <cursorShape>PointingHandCursor</cursorShape>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="pixmap">
                          <pixmap>images/image 1.jpg</pixmap>
                         </property>
                         <property name="scaledContents">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                        <widget class="QLabel" name="label_8">
                         <property name="text">
                          <string>San Felipe 4</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QFrame" name="frame_2">
                      <property name="maximumSize">
                       <size>
                        <width>301</width>
                        <height>1000</height>
                       </size>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_6">
                       <item>
                        <widget class="QLabel" name="label">
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>167</height>
                          </size>
                         </property>
                         <property name="cursor">
                          <cursorShape>PointingHandCursor</cursorShape>
                         </property>
                         <property name="mouseTracking">
                          <bool>true</bool>
                         </property>
                         <property name="focusPolicy">
                          <enum>Qt::ClickFocus</enum>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="pixmap">
                          <pixmap>images/image 1.jpg</pixmap>
                         </property>
                         <property name="scaledContents">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                        <widget class="QLabel" name="label_3">
                         <property name="text">
                          <string>San Felipe 1</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QFrame" name="frame_3">
                      <property name="maximumSize">
                       <size>
                        <width>301</width>
                        <height>1000</height>
                       </size>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_7">
                       <item>
                        <widget class="QLabel" name="label_2">
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>167</height>
                          </size>
                         </property>
                         <property name="cursor">
                          <cursorShape>PointingHandCursor</cursorShape>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                         <property name="pixmap">
                          <pixmap>images/image 1.jpg</pixmap>
                         </property>
                         <property name="scaledContents">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>San Felipe 2</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="frameButtons" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color:white;
font-size:12px;
padding-left:20px;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>INSTRUCTION: Select from the above, if it is existing. If not Click the NEW to generate a new road.</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QFrame" name="frame_6">
            <property name="styleSheet">
             <string notr="true">#btnCancel,#btnNew{
border:2px solid white;
border-radius:15px;
color:white;
font-size:14px;
}
#btnConfirm{
border:2px solid gray;
border-radius:15px;
color:gray;
font-size:14px;
}
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>20</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnCancel">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>36</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnNew">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>36</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>New</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnConfirm">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>36</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Confirm</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
